pkg/pathutil/pathutil_test.go

```diff
@@ -55,17 +56,54 @@ func TestIsSubPathExtending(t *testing.T) {
 				basePath:      filepath.Join("a", "b"),
 				extendingPath: filepath.Join("a", "a", "c"),
 			},
-			wantExtending:   nil,
-			wantIsExtending: false,
 		},
 		{
 			name: "is not smaller extending",
 			args: args{
 				basePath:      filepath.Join("a", "b"),
 				extendingPath: filepath.Join("a"),
 			},
-			wantExtending:   nil,
-			wantIsExtending: false,
+		},
+		{
+			name: "base path is not clean",
+			args: args{
+				basePath:      filepath.Join("d", "..", "a", "b", "..", "..", "a"),
+				extendingPath: filepath.Join("a", "b"),
```

Comment:
nit: test cases are hard to read
do you mind quickly replacing these with `filepath.FromSlash()`
maybe not needed so much I'm not sure

Reply:
Oh it's a good idea and easily done.